sources/Engine/Modules/Graphics/OpenGL/ShaderType.h

```diff
@@ -0,0 +1,12 @@
+#ifndef STARWIND_SHADERTYPE_H
```

Comment:
Other engine files contains `#pragma once` instead of include guards. So it will be better to have consistent sources.